qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -204,7 +205,7 @@ class FixedFrequencyTransmon(BasisGateLibrary):
     the sx and sy pulses. All pulses share the same duration and σ parameters.
     """
 
-    __default_values__ = {"duration": 160, "amp": 0.5, "β": 0.0}
+    __default_values__ = {"duration": 160, "amp": 0.5, "β": 0.0, "angle": 0.0}
```

Comment:
Even though there are no calibrations in this repo now that update the angle of cross-resonance pulses, I think we should still add `angle` and `tgt_angle` parameters to the cross-resonance basis gate libraries below along with this change to the single qubit library.